PAWS/Study Information/EnrollmentGroup.swift

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the PAWS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import Spezi
+import SpeziAccount
+import SpeziFirebaseAccount
+import SwiftUI
+
+
+@Observable
+class EnrollmentGroup: Module, EnvironmentAccessible {
+    @ObservationIgnored @Dependency var accountConfiguration: FirebaseAccountConfiguration?
+    @ObservationIgnored @Environment(Account.self) var account
+    var dateOfBirth: Date?
+    
+    var studyType: StudyType? {
+        guard let enrollmentDate = Auth.auth().currentUser?.metadata.creationDate,
+              let dateOfBirth,
+              let yearsOfAge = Calendar.current.dateComponents([.year], from: dateOfBirth, to: enrollmentDate).year else {
+            return nil
+        }
+        return yearsOfAge >= 18 ? .adult : .pediatric
+    }
+        
+    func configure() {
+        guard Auth.auth().currentUser != nil else {
+            return
+        }
+        Task {
+            self.dateOfBirth = await account.details?.dateOfBrith
```

Comment:
As noted above; this code likely crashes at runtime? Did you have any success to get the date of birth that way?

Reply:
No success: I just tried it, and `self.dateOfBirth` is `nil`.